### PR TITLE
Add Webpack 5 Support

### LINKED ISSUE #121: "this.exec is not a function" error in Webpack 5
When webfonts-loader is used with Webpack 5 (that has been released recently), the loader emits an error "this.exec is not a function" and the Webpack build fails.

### PR DESCRIPTION
**Pull request by kelunik**
Fixes #121.

I'm not sure whether the `url` option to `css-loader` can be done automatically without making the configuration more complex.

### CODE REVIEW

File: index.js [hunk 1]
```diff
@@ -193,6 +201,24 @@ module.exports = function (content) {
...
+    cssLoaderOptions.url = (url, resourcePath) => {
+      if (resourcePath === this.resourcePath) {
+        return false;
+      }
+
+      return cssLoaderUrlOption;
+    };
+
+    this.loaders[cssLoaderIndex].options = cssLoaderOptions;
+  }
```
Comment (evilebottnawi) [inline index.js:220]: Bad idea

Comment (kelunik) [reply index.js:220]: I guess it should be done in the pitch phase, but other than that, what's so bad about it?

Comment (andersk) [reply index.js:220]: Reaching into the configuration object of another module and mutating it is generally unexpected and fragile. The user might be using the `url` option for something else, or the object might be aliased elsewhere in the configuration. Or your heuristic might work half of the time and leave the user wondering why one thing works and another thing fails. Better to document what the user needs to configure.

Comment (andersk) [reply index.js:220]: Also, if nothing else, this configuration change has little to do with Webpack 5 support; the same issue is present with Webpack 4. So a separate PR would be better.

Comment (kelunik) [reply index.js]: I noticed I can just use `url: () => false`, as the `css-loader` is in a rule specific to `font.js` files.

**Review by andersk — APPROVED** [review]

**Comment by jeerbl** [comment]
@kelunik why do we need `url: false` in this? What does it change and why is it needed now

**Comment by kelunik** [comment]
@jeerbl It's required now, because the default in css-loader changed and I updated css-loader. It tries to resolve all URLs now, but there's no source file for the files it tries to resolve, because they're generated files. Without the option, it simply bails out.

**Comment by jeerbl** [comment]
Ok can't you just put `url: false` instead of a function?

**Comment by kelunik** [comment]
Maybe, I didn't try. :-)

File: README.md [hunk 2]
```diff
@@ -28,7 +28,12 @@ Add this rule to your Webpack config:
   test: /\.font\.js/,
   use: [
     MiniCssExtractPlugin.loader,
-    'css-loader',
+    {
+      loader: 'css-loader',
+      options: {
+        url: () => false
```
Comment (jeerbl) [inline README.md:34]: Change to `url: false` everywhere

Comment (kelunik) [reply README.md]: Done.

**Pull request merged**